contracts/utils/PausableAdminUpgradeable.sol

```diff
@@ -0,0 +1,170 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+import "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
+
+import "../interfaces/IPausableAdmin.sol";
+import "./PausableAdminErrors.sol";
+import "./PendingOwnableUpgradeable.sol";
+
+abstract contract PausableAdminUpgradeable is
+    PendingOwnableUpgradeable,
+    PausableUpgradeable,
+    IPausableAdmin
```

Comment:
Should probably import `IPausableAdminUpgradeable` (even if it would be the exact same interface)